tests/unit/models/primo.js

```diff
@@ -617,7 +618,120 @@ describe("Primo Model >", function() {
     });
   });
 
-  describe("primo model buildTemplate method >", function() {
+  describe("primo model isDefaultCoverImage method >", function() {
+    it("should return false when an actual coverImageUrl is returned by the API", function() {
+      var coverImageUrl = "https://assets.thirdiron.com/images/covers/0028-4793.png";
+      expect(primo.isDefaultCoverImage(coverImageUrl)).toEqual(false);
+    });
+
+    it("should return true when a default coverImageUrl is returned by the API", function() {
+      var coverImageUrl = "https://assets.thirdiron.com/images/covers/default.png";
+      expect(primo.isDefaultCoverImage(coverImageUrl)).toEqual(true);
+    });
+  });
+
+  describe("primo model getDirectToPDFUrl method >", function() {
+    it("should not include a fullTextFile property in the BrowZine API response for a journal", function() {
+      var scope = {
+        result: {
+          pnx: {
+            display: {
+              type: ["journal"]
+            },
+
+            addata: {
+              issn: ["0096-6762", "0028-4793"]
+            }
+          }
+        }
+      };
+
+      var data = primo.getData(journalResponse);
+
+      expect(data).toBeDefined();
+
+      expect(primo.getDirectToPDFUrl(scope, data)).toBeNull();
+    });
+
+    it("should include a fullTextFile property in the BrowZine API response for an article", function() {
```

Comment:
Same thing here - 
describe("primo model getDirectToPDFUrl method >", function() {
  describe("given an API response for a journal", ... 
  );
  describe("given an API response for an article", ...
    var scope = { ... doi: ["10.1136/bmj.h2575" ...
    it("should include a fullTextFile property", ...
```

Thoughts?

Reply:
Thanks Karl! See, https://github.com/thirdiron/browzine-discovery-service-adapters/pull/45#discussion_r208045501